tools/c7n_mailer/c7n_mailer/record.py

```diff
@@ -13,6 +13,14 @@
 # limitations under the License.
 import json
 from cStringIO import StringIO
+from dateutil import parser
+from pytz import timezone
+
+
+def date_time_format(utc_str, tz_str='US/Eastern', format='%Y %b %d %H:%M %Z'):
```

Comment:
What would the dateutil line look like to convert the UTC datetime object to a different timezone when it's rendered as a string?